cmux-tui/bindings/reconcile_registry_artifact.py

```
@@ -0,0 +1,307 @@
+from __future__ import annotations
+
+import argparse
+import base64
+import hashlib
+import json
+import os
+from pathlib import Path
+import subprocess
+import sys
+import time
+from typing import Any, Callable, Optional, Sequence
```

Ruff (0.16.0)

[warning] Import from `collections.abc` instead: `Callable`, `Sequence`

Import from `collections.abc`

(UP035)